invenio_rdm_records/oai.py

```diff
@@ -40,6 +41,16 @@ def oai_marcxml_etree(pid, record):
     return etree.fromstring(MARCXMLSerializer().serialize_object(record["_source"]))
 
 
+def oai_dcat_etree(pid, record):
+    """OAI DCAT-AP format for OAI-PMH.
+
+    It assumes that record is a search result.
+    """
+    return etree.fromstring(
+        DCATSerializer().serialize_object(record["_source"]).encode(encoding="utf-8")
```

Comment:
🚀 very nice, I opened and issue to do this in other functions that don't 💪  https://github.com/inveniosoftware/invenio-rdm-records/issues/1255